app/controllers/people_controller.rb

```diff
@@ -0,0 +1,54 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+  def show
+    id = params[:id]
+    @person = Person.find(id)
+  end
+
+  def tasks
+    id = params[:id]
+    @tasks = Person.find(id).sorted_tasks
+  end
+
+  def new
+    @person = Person.new
+    @title = "Add"
+    @action = "create"
+  end
+
+  def create
+    Person.create(person_params[:person])
+
+    redirect_to "/people"
+  end
+
+  def destroy
+    Person.destroy(params[:id])
+
+    redirect_to "/people"
+  end
+
+  def edit
+    id = params[:id]
+    @person = Person.find(id)
+    @title = "Modify"
+    @action = "update"
+
+    render :new
+  end
+
+  def update
+    Person.update(params[:id], person_params[:person])
+
+    redirect_to "/people"
+  end
+
+  private
+
+  def person_params
+    params.permit(person:[:name, :bio])
```

Comment:
If we change this to `params.require(:person).permit(:name, :bio)` we can actually omit the `[:person]` part on the above lines that call `person_params`.